localstack/services/events/provider.py

```diff
@@ -199,6 +200,30 @@ def create_connection(
 
         return call_moto(context)
 
+    def put_targets(
```

Comment:
The original issue also mentioned an upper limit of 64 characters. Did you look into this as well? :thinking: